batch/2017-01-01.4.0/examples/TaskAddCollection_Complex.json

```diff
@@ -0,0 +1,76 @@
+{
+  "parameters": {
+    "api-version": "2017-01-01.4.0",
+    "jobId": "jobId",
+    "client-request-id": "00000000-0000-0000-0000-000000000000",
+    "ocp-data": "Fri, 17 Feb 2017 00:00:00 GMT",
+    "taskCollection": {
+      "value": [
+        {
+          "id": "complex1",
+          "commandLine": "cmd /c dir /s",
+          "resourceFiles": [
+            {
+              "blobSource": "https://account.blob.core.windows.net/",
+              "filePath": "file1"
+            }
+          ],
+          "environmentSettings": [
+            {
+              "name": "env1",
+              "value": "value1"
+            },
+            {
+              "name": "env2",
+              "value": "value2"
+            }
+          ],
+          "affinityInfo": {
+            "affinityId": "affinityId"
+          },
+          "constraints": {
+            "maxWallClockTime": "P1D",
+            "retentionTime": "P2D",
+            "maxTaskRetryCount": 5
+          },
+          "multiInstanceSettings": {
+            "numberOfInstances": 3,
+            "coordinationCommandLine": "cmd /c echo coordinating",
+            "commonResourceFiles": [
+              {
+                "blobSource": "https://common.blob.core.windows.net/",
+                "filePath": "common.exe"
+              }
+            ]
+          }
+        },
+        {
+          "id": "simple3",
+          "commandLine": "cmd /c dir /s"
+        }
+      ]
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "status": "success",
+            "taskId": "simple3",
```

Comment:
Maybe put taskId before status as this will make it clearer what this structure relates to?  Perhaps also retain the same order as the request, though I know we do not guarantee that and there is something to be said for showing that we don't!